python/Scripts/mxvalidate.py

```diff
@@ -0,0 +1,394 @@
+#!/usr/bin/env python
+'''
+Verify that a specified file is a valid MaterialX document.
+
+Doug Smythe
```

Comment:
For the MaterialX repository, we've dropped attribution lines for each script and source file, since GitHub can track authorship at a finer granularity (e.g. multiple authors contributing to different sections of a single file).

Reply:
done